app/src/main/java/com/github/orkest/Model/Post.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.orkest.Model
+
+data class Post(var user: User,
+                var postDescription: String,
+                var song: Song,
```

Comment:
Instead of storing a whole user for a post,  just the string username would suffice since we're only displaying that on the post. When clicked on, we would want to display the corresponding profile and the profileViewActivity only needs the username to do so.

Reply:
You are right but I would also need the profile picture Id too